amnestywebsite/humanity-theme
# Remove social share pattern

Ref: https://github.com/amnestywebsite/humanity-theme/issues/201, https://github.com/amnestywebsite/humanity-theme/issues/182

**Steps to test**:
1. checkout branch
2. create a new post
3. open the inserter
4. there should be no patterns listed
